plugin-gradle/README.md

```diff
@@ -139,10 +139,10 @@ To apply freshmark to all of the `.md` files in your project, with all of your p
 ```gradle
 spotless {
 	freshmark {
-		target 'README.md', 'CONTRIBUTING.md'	// defaults to '**/*.md'
-		propertiesFile('gradle.properties')		// loads all the properties in the given file
+		target 'README.md', 'CONTRIBUTING.md'			// defaults to '**/*.md'
+		propertiesFile 'fresh.xml', 'mark.properties'	// loads all the properties in the given files
 		properties {
-			it.put('key', 'value')				// specify other properties manually
+			it.put('key', 'value')						// specify other properties manually
 		}
```

Comment:
I prefer the old example.  It's standard to have a `gradle.properties` file which contains project versions which are useful for markdown formatting.  It's great to have the new shared parser, but I think the example should remain the **typical** usecase, rather than an exotic one.

The text can say something like "multiple properties files can be loaded, in any format supported by FormatterProperties." with a link to the FormatterProperties javadoc.